File: LAB 19/Task3.cpp
#include <iostream>
using namespace std;

// Function to calculate factorial using recursion
int factorial(int n) {
    if (n == 0 || n == 1)
        return 1;  // Base case
    else
        return n * factorial(n - 1);  // Recursive case
}

int main() {
    int num;
    cout << "Input: ";
    cin >> num;

    int result = factorial(num);

    cout << "Output: Factorial = " << result << endl;

    return 0;
}

File: LAB 19/Task3.py
def factorial(n):
    # Base case
    if n == 0 or n == 1:
        return 1
    else:
        # Recursive case
        return n * factorial(n - 1)


# Ask the user to enter a number
num = int(input("Input: "))

# Calculate factorial
result = factorial(num)

# Display output in required format
print("Output: Factorial =", result)
